Update business card experiment configurations

rukizhazard opened the pull request:
- Images
  - 512x512: https://airocrhbi.blob.core.windows.net/ocr-data/Vertical/vdipainter/logs/business_card/super_res/2022-09-29T09-27-08_txtinpaintbizcard512_fsdp-fairscale/images/train/
  - 128x128:: https://airocrhbi.blob.core.windows.net/ocr-data/Vertical/vdipainter/logs/business_card/2022-09-20T04-14-37_txtinpaintbizcard128_fsdp-fairscale-t5xl/images/train/
- AML experiment
  - https://ml.azure.com/experiments/id/2b4624f9-701b-4daa-b4b5-98fa6c7ced77/runs/vdiptbizcard_1664443461_91040a6d?wsid=/subscriptions/b8da985a-830d-4d20-b9e5-8d4c0d798c7f/resourcegroups/FY20Vision/workspaces/OCR2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

rukizhazard committed:
Update business card experiment configurations

rukizhazard committed:
- Remove unused comments
- Add newline at the end of files

rukizhazard committed:
Remove unused comment

taoisu commented on cloud/aml_submit.py:
```diff
@@ -41,6 +42,8 @@ def main(
         'train_avid_sr_128x512_effinet': 'cloud/entry_avid_train_sr_128x512-effinet.sh',
         'train_avid_sr_128x512_opainet': 'cloud/entry_avid_train_sr_128x512-opainet.sh',
         'train_avid_inpaint_128_t5xxl': 'cloud/entry_avid_train_inpaint_128-t5xxl.sh',
+        'train_bizcard_inpaint_128_t5xxl': 'cloud/entry_bizcard_train_inpaint_128-t5xxl.sh',
```
Please rename the name since it's not inpaint

rukizhazard replied:
Fixed, thank you

taoisu commented on cloud/aml_submit.py:
```diff
@@ -41,6 +42,8 @@ def main(
         'train_avid_sr_128x512_opainet': 'cloud/entry_avid_train_sr_128x512-opainet.sh',
         'train_avid_inpaint_128_t5xxl': 'cloud/entry_avid_train_inpaint_128-t5xxl.sh',
+        'train_bizcard_inpaint_128_t5xxl': 'cloud/entry_bizcard_train_inpaint_128-t5xxl.sh',
+        'train_bizcard_inpaint_512': 'cloud/entry_bizcard_train_inpaint_512.sh',
```
Please rename, something like `bizcard_sr_512x4`

rukizhazard replied:
Fixed, thank you

taoisu commented on cloud/entry_bizcard_train_inpaint_128-t5xxl.sh:
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Please rename this file

rukizhazard replied:
Fixed, thank you

taoisu commented on cloud/entry_bizcard_train_inpaint_512.sh:
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Please rename this file

rukizhazard replied:
Fixed, thanks

taoisu commented on configs/just-diffusion/bizcard128_fsdp-fairscale-t5xl.yaml:
```diff
@@ -0,0 +1,69 @@
+model:
```
Please rename this file as you're not using t5xl for conditioning

taoisu replied:
maybe using fairscale-uncond

rukizhazard replied:
Sure, thanks

taoisu commented on configs/just-diffusion/bizcard512_fsdp-fairscale.yaml:
```diff
@@ -0,0 +1,80 @@
+model:
```
Suggest bizcard_sr_512x4-fairscale

rukizhazard replied:
Fixed, thanks

taoisu commented on scripts/load_pickle.py:
```diff
@@ -0,0 +1,9 @@
+import pickle
```
Do we need this file?

rukizhazard replied:
May not be necessary, but for checking pickle file content? Would you prefer removing it?

rukizhazard replied:
Removed, thanks

rukizhazard committed:
- Refine namings
  - bizcard512_fsdp-fairscale > bizcard_sr_512x4-fairscale
  - bizcard128_fsdp-fairscale-t5xl.yaml > bizcard_128-fairscale-uncond
  - entry_bizcard_train_inpaint_512 > entry_bizcard_train_sr_512x4
  - entry_bizcard_train_inpaint_128-t5xxl > entry_bizcard_train_128-uncond

rukizhazard committed:
Resolve comments
